src/app/pages/psbt-request/components/psbt-request-details.tsx

```diff
@@ -21,7 +23,11 @@ export function PsbtRequestDetails({ details, payloadTxHex }: PsbtRequestDetails
             <pre>
               {JSON.stringify(
                 details,
-                (_, value) => (typeof value === 'bigint' ? value.toString() : value),
+                (_, value) => {
+                  if (typeof value === 'bigint') return value.toString();
+                  if (isTypedArray(value)) return Buffer.from(value).toString('hex');
```

Comment:
Much of the efforts of the signer library/Jannik in Stacks.js have been moving to `UInt8Array` to remove the dependence on `Buffer`.

Use the `bytesToHex` utility fn directly on value, rather first turning to buffer.

Reply:
Would also recommend making a function `parseJsonReadable` or something, rather than putting this logic in template

Reply:
Yeah, was thinking about this and realized I didn't use the `bytesToHex` ...def know that is the better choice here. I'll fix.